modules/dhcp_common/server.rb

```diff
@@ -185,7 +185,7 @@ def clean_up_add_record_parameters(in_options)
       subnet = options.delete("network")
 
       # convert string keys to symbols
-      options = options.inject({}){|all, current| all[current[0].to_sym] = current[1]; all}
+      options = options.each_with_object({}){|current, all| all[current[0].to_sym] = current[1]; }
```

Comment:
```suggestion
      options = options.each_with_object({}){|current, all| all[current[0].to_sym] = current[1] }
```

Reply:
Wait, itsnt this a change? It returned `all`, now it returns what?

Reply:
https://www.rubocop.org/en/stable/cops_style/#styleeachwithobject describes exactly this: "...without the need to return the object at the end."